docs/specs/schema.yml

```
@@ -522,3 +522,31 @@ outputs:
   b.json:
   c.json: |
     {"metadata": {"href": "b"}}
+---
+# html post process for `markdown` properties
+inputs:
+  docfx.yml:
+  a.yml: |
+    #YamlMime:TestData
+    markdown: |
+      # h1
+      <script>alert(0);</script>
+      <link rel="stylesheet" href="https://a.com/b.css" />
+      <div style="display:hidden"></div>
+      <style>h1 {color:red;}</style>
+  _themes/ContentTemplate/schemas/TestData.schema.json: |
+    {
+      "properties": {
+        "markdown": {
```

can omit `type`, the suggested contentType should be `markdown` but somehow `Markdown` also works due to how JSON.NET handle enums.

```suggestion
        "markdown": { "contentType": "markdown" }
```